src/bgw/scheduler.c

```diff
@@ -481,6 +481,10 @@ ts_update_scheduled_jobs_list(List *cur_jobs_list, MemoryContext mctx)
 		else if (cur_sjob->job.fd.id > new_sjob->job.fd.id)
 		{
 			scheduled_bgw_job_transition_state_to(new_sjob, JOB_STATE_SCHEDULED);
+			elog(DEBUG1,
+				 "sjob %d was new, its fixed_schedule is %d",
+				 new_sjob->job.fd.id,
+				 new_sjob->job.fd.fixed_schedule);
```

Comment:
If you want to keep this as debug output, it might be good to add more information as details and use `ereport`.

Reply:
+1